src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/manual/ManualSyncRunner.java

```diff
@@ -120,28 +122,36 @@ private File searchForServerElements(Project project, Element element, ProgressS
             nodeIds.add(element.getLocalID());
         }
         File sendData = MMSUtils.createEntityFile(this.getClass(), ContentType.APPLICATION_JSON, nodeIds, MMSUtils.JsonBlobType.SEARCH);
+        //TODO: Return to recursive search by rootElements / ownerID it was crashing MMS4 Elasticsearch
```

Comment:
This `TODO` is no longer valid as we have not changed from using search, pending a server-side resolution.